src/System.Private.CoreLib/shared/System/Resources/FastResourceComparer.cs

```diff
@@ -70,11 +71,11 @@ public bool Equals(string a, string b)
             return string.Equals(a, b);
```

Comment:
This should accept nulls right?

Reply:
yes and no, this implements `IEqualityComparer`, wile `Equals` is fine with nulls the `GetHashCode` is not - there is no way I know of to express that with current syntax

Reply:
> there is no way I know of to express that with current syntax

We should annotate it to say that nulls are accepted and include a TODO-NULLABLE comment for being able to annotate this appropriately.